config/nxp/chip-module/Kconfig.features

```diff
@@ -24,11 +24,11 @@ config CHIP_WIFI
 	default y
 	select WIFI
 	select WIFI_NXP
-	select WPA_SUPP
```

Comment:
I suppose this change is not the result of adding the driver to the Zephyr but rather the refactoring needed to align everything? By that I mean that the WPA_SUPP_AP was defined in the hostap not in the driver (and the hostap have been integrated in Zephyr for a while already). Anyway, I wonder if we should first upstream the patch that affects platforms other than nrfconnect. @LuDuda, @kkasperczyk-no any thoughts here?

Reply:
nR70 now uses hostap from the upstream where the renaming was done.

Reply:
Yes, my only concern was if the NXP already used the hostap from Zephyr upstream that includes this change.